### DISCUSSION TITLE
Implement overlaying images

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#147] [Ideas]
Hi Magne, i think it would be very useful to implement the overlaying of images like maps or aerial pictures, to let the users copy the data they contain, like unit positions and movements (maps) and old roads, fortifications and bunkers (photos).
Do you think it's possible?
Regards
Federico

**Comment by kjellmf** [comment]
This should be possible. See https://viglino.github.io/ol-ext/examples/layer/map.geoimage.html for an example.

**Comment by kjellmf** [comment]
I've added an issue for this (#149)

**Comment by kjellmf** [comment]
I have now added basic support for overlaying images.
